src/ppx/dune

```diff
@@ -13,10 +13,10 @@ let () = Jbuild_plugin.V1.send @@ {|
  (public_name lwt_ppx)
  (synopsis "Lwt PPX syntax extension")
  (modules ppx_lwt)
- (libraries compiler-libs.common ocaml-migrate-parsetree ppx_tools_versioned)
+ (libraries compiler-libs.common ppxlib)
```

Comment:
AFAICT we are now using `ocaml-compiler-libs.common` (e.g. new line 268).

Reply:
done